### ISSUE TITLE
NVDA doesn't report text form element placeholder when the element has a label

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5403]
**Reported by leonarddr on 2015-10-07 11:07**
This problem can be reproduced using the attachment below. When using tab to navigate through a form, NVDA reports the placeholder of a text element, but only when it has no label. When the element has a label attached, the placeholder is ignored. Also, the placeholder is being ignored entirely when navigating using arrow keys in browse mode.

**Comment by jcsteh** [comment]
Duplicate of #1638.
